modules/scripts/startup-script/README.md

```diff
@@ -138,7 +138,7 @@ sudo journalctl -u google-startup-scripts.service
   settings:
     runners:
       # Some modules such as filestore have runners as outputs for convenience:
-      - $(homefs.install_nfs_client_runner)
```

Comment:
I am not convinced that trading
```
$(homefs.install_nfs_client_runner)

for

((module.homefs.network_storage.client_install_runner))
```

is better. Is there any way we could use `$(homefs.network_storage.client_install_runner)`. We generally not favored users using the `(())` syntax. Can you add some justification as to why this change is improving the user experience.

Reply:
Addressed by using newly supported $() functionality.